docs/streams/developer-guide/processor-api.html

```diff
@@ -351,6 +352,21 @@ <h2>
                     your store.</p>
             </div>
         </div>
+        <div class="section" id="accessing-processor-context">
+            <h2><a class="toc-backref" href="#id10">Accessing Processor Context</a><a class="headerlink" href="#accessing-processor-context" title="Permalink to this headline"></a></h2>
+            <p><code class="docutils literal"><span class="pre">ProcessorContext</span></code> gives access to metadata related with Processor state and the current record being processed.</p>
```

Comment:
Add one more sentence at the beginning: `As we have mentioned in <href=#defining-a-stream-processor>Defining a Stream Processor<\href>, a <code>ProcessorContext</code> to control the processing workflow, such as scheduling a punctuation function, and committing the current processed state. This object can also be used to access the metadata related with...`